libs/community/langchain_community/document_compressors/flashrank_rerank.py

```diff
@@ -26,8 +26,12 @@ class FlashrankRerank(BaseDocumentCompressor):
     """Flashrank client to use for compressing documents"""
     top_n: int = 3
     """Number of documents to return."""
+    score_threshold: float = 0.0
+    """Minimum relevance threshold to return."""
     model: Optional[str] = None
     """Model to use for reranking."""
+    prefix_metadata: Optional[str] = None
```

Comment:
Set it to empty string by default  otherwise it will break when it's not defined.
```suggestion
    prefix_metadata: Optional[str] = ""
```